tests/test_bot.py

```diff
@@ -2155,7 +2155,7 @@ async def test_forward_messages(self, bot, chat_id):
         msg1, msg2 = await tasks
 
         forward_messages = await bot.forward_messages(
-            chat_id, from_chat_id=chat_id, message_ids=(msg1.message_id, msg2.message_id)
+            chat_id, from_chat_id=chat_id, message_ids=sorted((msg1.message_id, msg2.message_id))
```

Comment:
any idea why the bot api wants them to be sorted?

Reply:
> The identifiers must be specified in a strictly increasing order.

https://core.telegram.org/bots/api#forwardmessages

don't know *why* they want that, though …

Reply:
should PTB auto sort the passed message ids?

Reply:
mh … my gut says no. Sroting can be expensive and if the user already takes care of it, it's wasted effort. then again it's only 100 entries max and the effort spent on sorting an already sorted list is probably small 🤔

Reply:
well the performance doesn't matter, since the user can do it in their code, or we do it library side. It will take "time" either way. The advantage is the user doesn't have to worry about it

Reply:
Performance does matter if thr user always passes sorted lists - then it's unnecessary overhead. I'd like to see some small benchmark before going ahead with this. In any case, it would be a separate pr :)